.travis.yml

```diff
@@ -1,6 +1,4 @@
 language: python
-services:
```

Comment:
I think this needs to be included. But if there is any solid reason for removing this, then it's ok.